app/mseed_converter.py

```diff
@@ -31,3 +35,23 @@ def convert_to_mseed(azure_storage_decimated_file_path: str, file_format: str):
     except subprocess.CalledProcessError as e:
         logger.warning(e.stderr)
         raise Exception(e.stderr)
+
+
+def poll_convert_queue():
+    logger.info("started polling convert queue ...")
+    msg: QueueMessage
+    while True:
+        msg = convert_queue.fetch_message()
+        if msg:
+            logger.info(f"new msg arrived in convert queue! {msg}")
+            message_content = json.loads(msg.content)
+            azure_storage_decimated_file_path = message_content["path"]
+            file_format = message_content["format"]
+            convert_to_mseed(azure_storage_decimated_file_path, file_format)
+            convert_queue.delete_message((msg))

```

Comment:
```suggestion
            convert_queue.delete_message(msg)

```
Or is there a reason? Black should have caught that